src/routes_initializer.coffee

```diff
@@ -17,12 +17,14 @@ class RoutesInitializer
 
   constructor: (@routesFile, @reverseRoutes, @addRoute) ->
     @_routes = []
+    @_scope = ''
     require(@routesFile) @getRouteDirectives()
 
 
   # Returns a route directive for a specific http method to be called in a routes file
   getHttpDirective: (httpMethod) ->
     (url, {to, as}) =>
+      url = "#{@_scope}#{url}"
```

Comment:
On second thought normalizing urls probably should be handled in all cases and not just scoping. Let's use `url = "#{@_scope}/#{url}"` for this PR and we can open a new PR for normalizing paths if express doesn't already do this for us